### PR TITLE
Implements Actor Script direction testing

### PR DESCRIPTION
**Pull request by thomas-alrek**
This feature implements a Actor script for conditionally testing a Actors current direction.
This pull request partially solves issue #33

### CODE REVIEW

File: src/lib/compiler/eventTypes.js [hunk 1]
```diff
@@ -71,6 +72,7 @@ export const EventNames = {
   [EVENT_IF_VALUE]: "If Variable Has Value",
   [EVENT_IF_INPUT]: "If Joypad Input Pressed",
   [EVENT_IF_ACTOR_AT_POSITION]: "If Actor At Position",
+  [EVENT_IF_ACTOR_DIRECTION]: "If Actor Direction matches",
```
Comment (chrismaltby) [inline src/lib/compiler/eventTypes.js]: Could you please change this to "If Actor Facing Direction"

**Comment by chrismaltby** [comment]
Nice one! Works perfectly in all versions and matches on code style too. 💯

Do you mind just switching the label as requested in my comment to be "If Actor Facing Direction", then I can merge this.

**Comment by thomas-alrek** [comment]
@chrismaltby Done, I have now renamed the label

File: src/lib/compiler/eventTypes.js (hunk 1 above)
Comment (thomas-alrek) [reply src/lib/compiler/eventTypes.js]: Done 🙂

**Comment by chrismaltby** [comment]
Awesome! Thanks! Just to let you know, the event names are now coming from `/src/lang/en.json` to support localisation to new languages, I've added this label in commit 3ffdd595acca86e67a133d21dfa76657eac9310c.

**Pull request merged**